__tests__/integration.test.js

```diff
@@ -52,4 +52,17 @@ describe("/api/topics", () => {
         });
     });
 
+})
+
+describe('Being on api',()=>{
```

Comment:
this isn't the best test description, I would just put the actual endpoint in the describe block ie. "/api"